src/components/Error/ErrorMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import CloseButton from '../Button/CloseButton';
+
+interface Props {
```

Comment:
Bit generic, specify what it's for

Reply:
Should be something like `ErrorMessageProps`